app/modules/site_settings/models.py

```diff
@@ -313,6 +349,24 @@ def set_key_value(cls, key, value):
 
             SocialGroup.site_settings_updated()
 
+        if cls.HOUSTON_SETTINGS[key].get('isApiKey', False):
+            # generate output file as
+            output = [
+                f"export const transloaditKey = {cls.get_value('transloaditKey')};",
+                f"export const transloaditTemplateId = {cls.get_value('transloaditTemplateId')};",
+                f"export const transloaditService = {cls.get_value('transloaditTemplateId')};",
+                f"export const googleMapsApiKey = {cls.get_value('googleMapsApiKey')};",
+                f"export const sentryDsn = {cls.get_value('sentryDsn')};",
+                f"export const flatfileKey = {cls.get_value('flatfileKey')};",
+            ]
+
+            # write to appropriate apikeys.js depending on if we're codex or mws
+            from flask import current_app
+
+            out_filename = f"_frontend.{current_app.config.get('PROJECT_NAME').lower()}/src/constants/apiKeys.js"
```

Comment:
We want a view to produce this information. The frontend source is not available to be written to in a production deployment.

Reply:
In other words, frontend devs will change where they get these keys based on where we put the view or where it is already available.